django_project/base/templates/account/signup.html

```diff
@@ -1,33 +1,12 @@
 {% load staticfiles %}
 {% load i18n %}
-{% load account socialaccount %}
```

Comment:
Why did you remove these lines?

Reply:
I think this lines is needed, please bring it back